lms/static/sass/views/_account-settings.scss

```diff
@@ -36,45 +38,254 @@
 .wrapper-account-settings {
 
     .wrapper-header {
+        max-width: grid-width(12);
+        height: 139px;
+        border-bottom: 4px solid $m-gray-l4;
 
         .header-title {
             @extend %t-title4;
             margin-bottom: ($baseline/2);
+            padding-top: ($baseline*2);
         }
 
         .header-subtitle {
             color: $gray-l2;
         }
+
+        .account-nav {
+            @include float(left);
+            margin: ($baseline/2) 0;
+            padding: 0;
+            list-style: none;
+
+            .account-nav-item {
+                @include float(left);
+                display: flex;
+                margin: 0;
+                text-transform: none;
+                justify-content: center;
+
+                .account-nav-link{
+                    font-size: em(16);
+                    color: $dark-gray;
+                    padding: 5px 25px 23px;
+                    display: inline-block;
+                    border-radius: 0;
+                }
+                .active{
+                    border-bottom: 4px solid $courseware-border-bottom-color !important;
+                }
+            }
+            li, div {
+                display: inline-block;
+                margin: 0 $baseline+1 0 0;
+                font-size: em(14);
+                font-weight: $font-semibold;
+                text-transform: uppercase;
+                letter-spacing: 0 !important;
+
+                &:last-child {
+                    margin-right: 0;
+                }
+
+                button {
+                    @extend %ui-clear-button;
+                    @extend %btn-no-style;
+                    @include appearance(none);
+                    display:block;
+                    padding: ($baseline/4);
+                    color: $link-color;
+                    font-weight: $font-semibold;
+
+                    &:hover, &:focus, &:active {
```

Comment:
There is no visible hover focus or active state defined here.  When I tried to use this control with a keyboard I could not determine when these tabs had focus.  When using a mouse, there was no hover state.  Be sure not to use color alone to indicate focus.  I recommend removing outline:none.